syntax = "proto3";

import "hq/dto/BasicQot.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.response";
option java_outer_classname = "BasicQotResponseProto";
message BasicQotResponse {
      // 基础行情返回
      repeated BasicQot basicQot=1;
}
